src/ImageSequencerNode.js

```diff
@@ -0,0 +1,142 @@
+ImageSequencer = function ImageSequencer(options) {
```

Comment:
Hi, just checking - is this instead of another module, and how much redundancy is there? Or is this just a core module which is then included into another? I just want to be sure we don't replicate too much code if we could be using `require()` -- and it's been a few weeks :-P -- can you help me understand? Thank you!! Exciting to see work starting up!